src/components/Button/Button.test.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import Button from './Button';
+
+test('renders without crashing', () => {
+   const tree = renderer.create(<Button>Test Button</Button>).toJSON();
```

Comment:
should `.toJSON();` be on the next line like the rest of the things?

Reply:
Nope, this is how prettier formatted it :)
It only pushed it onto a new line if it can't fit on the first within the 80 char limit.